ccm_web/client/src/App.tsx

```diff
@@ -72,25 +72,25 @@ function App (): JSX.Element {
 
   return (
     <Layout {...{ features, pathnames }} devMode={globals?.environment === 'development'} csrfToken={csrfToken}>
-      <Switch>
-        <Route exact={true} path='/'>
+      <Routes>
+        <Route path='/' element={
           <Home globals={globals} csrfToken={csrfToken} course={course} setCourse={setCourse} getCourseError={getCourseError} />
-        </Route>
+          } />
         {features.map(feature => {
           return (
-            <Route key={feature.data.id} path={feature.route}>
+            <Route key={feature.data.id} path={feature.route} element={
               <feature.component
                 globals={globals}
                 csrfToken={csrfToken}
                 course={course}
                 title={feature.data.title}
                 helpURLEnding={feature.data.helpURLEnding}
               />
-            </Route>
+            }/>
           )
         })}
-        <Route><NotFound /></Route>
-      </Switch>
+        <Route path="/*" element={<NotFound />}/>
```

Comment:
can you tell me why `/*` is treated a unknown route by react router dom package? Can you suggest some documentation if it there? 
This works Perfectly!